datafusion/core/src/datasource/physical_plan/parquet/mod.rs

```diff
@@ -739,7 +739,17 @@ impl ExecutionPlan for ParquetExec {
     }
 
     fn statistics(&self) -> Result<Statistics> {
-        Ok(self.projected_statistics.clone())
+        // When filters are pushed down, we have no way of knowing the exact statistics.
+        // Note that pruning predicate is also a kind of filter pushdown.
+        let stats = if self.pruning_predicate.is_some()
+            || self.page_pruning_predicate.is_some()
```

Comment:
I think bloom filters should also belong in this list 🤔

Reply:
Bloom filters are used by `pruning_predicate `.